ceph.spec.in

```diff
@@ -131,6 +131,7 @@ BuildRequires:	python-pecan
 BuildRequires:	python-requests
 BuildRequires:	python-virtualenv
 BuildRequires:	python-werkzeug
+BuildRequires:	socat
```

Comment:
@smithfarm i am not Bassam. but i agree that socat is only used for tests. is there anything we can do different? if it's only for test?